op-batcher/batcher/driver.go

```diff
@@ -478,6 +521,32 @@ func (l *BatchSubmitter) safeL1Origin(ctx context.Context) (eth.BlockID, error)
 	return status.SafeL2.L1Origin, nil
 }
 
+// cancelBlockingTx creates an empty transaction of appropriate type to cancel out the incompatible
+// transaction stuck in the txpool. In the future we might send an actual batch transaction instead
+// of an empty one to avoid wasting the tx fee.
+func (l *BatchSubmitter) cancelBlockingTx(queue *txmgr.Queue[txID], receiptsCh chan txmgr.TxReceipt[txID]) {
+	txdata := txData{
+		frames: []frameData{
+			frameData{
+				data: []byte{},
+				id: frameID{
+					chID:        cancelID,
+					frameNumber: 0,
+				},
+			},
+		},
+	}
```

Comment:
... we wouldn't then need to create any dummy data here...

Reply:
looks like I still need some dummy data to create the tx candidate